changelog.adoc

```diff
@@ -54,6 +54,7 @@ and this project tries to adhere to https://semver.org/spec/v2.0.0.html[Semantic
 ** 'org.openapi.generator:6.6.0'
 ** 'org.spockframework:spock-core:2.3-groovy-3.0'
 * `dtcw` and `dtcw.ps1`:
+** Changed environment variable `DTC_ROOT` to be configurable specify the parent directory the installation directory, which defaults the value of the environment variable `HOME`
```

Comment:
I introduced `DTC_ROOT` as prerequisite to address #1062. Since `DTC_ROOT` was not accessible until now, I would change the description a bit. What about:

"Add new environment variable `DTC_ROOT` to make the docToolchain installation directory configurable. The default installation directory remains $`HOME/.docoolchain`.